### PR TITLE
app/vmagent: fix check of the DataDog agent path requests when requests have trailing slashes

### PR DESCRIPTION
**Pull request by dmitryk-dk**
Related issue: https://github.com/VictoriaMetrics/VictoriaMetrics/issues/5078

Signed-off-by: dmitryk-dk [d.kozlov@victoriametrics.com](mailto:d.kozlov@victoriametrics.com)

### CODE REVIEW

**Review by hagen1778 — APPROVED** [review]
@dmitryk-dk have you verified dd agent continues to work with vmagent after this change?

**Comment by dmitryk-dk** [comment]
> @dmitryk-dk have you verified dd agent continues to work with vmagent after this change?

I ran this command 
```
docker run -d --name dd-agent \
  -v /var/run/docker.sock:/var/run/docker.sock:ro \
  -v /proc/:/host/proc/:ro \
  -v /sys/fs/cgroup/:/host/sys/fs/cgroup:ro \
  -e API_KEY={my_api_key_here} \
  -e SD_BACKEND=docker \
  -e NON_LOCAL_TRAFFIC=false \
  datadog/docker-dd-agent:latest
  ```
  and changed the config file in the docker container. I saw the metrics in storage.
  
also, I checked the request via cURL to detect the problem when there is a situation described in this issue
https://github.com/VictoriaMetrics/VictoriaMetrics/pull/2670

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -63,6 +63,7 @@ The sandbox cluster installation is running under the constant load generated by
...
 * BUGFIX: [MetricsQL](https://docs.victoriametrics.com/MetricsQL.html): allow passing [median_over_time](https://docs.victoriametrics.com/MetricsQL.html#median_over_time) to [aggr_over_time](https://docs.victoriametrics.com/MetricsQL.html#aggr_over_time). See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/5034).
 * BUGFIX: [vminsert](https://docs.victoriametrics.com/Cluster-VictoriaMetrics.html): fix ingestion via [multitenant url](https://docs.victoriametrics.com/Cluster-VictoriaMetrics.html#multitenancy-via-labels) for opentsdbhttp. See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/5061). The bug has been introduced in [v1.93.2](https://github.com/VictoriaMetrics/VictoriaMetrics/releases/tag/v1.93.2).
+* BUGFIX: [vmagent](https://docs.victoriametrics.com/vmagent.html): fix check of the DataDog agent path requests when requests have trailing slashes. See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/5078).
```
[the remark this answers is not in the document]
Comment (dmitryk-dk) [reply docs/CHANGELOG.md]: fixed

**Review by valyala — APPROVED** [review]

**Comment by valyala** [comment]
@dmitryk-dk , thanks for the pull request!

**Pull request merged**